app/models/event.rb

```diff
@@ -16,16 +16,28 @@ class Event < ActiveRecord::Base
   # Validate slots if event allows signup
   # validates :slots, presence: true, if: :signup?
 
+  scope :start, -> { order(starts_at: :asc) }
   scope :calendar, -> { all }
   scope :nollning, -> { where(category: 'nollning') }
   scope :from_date, -> (date) { where('(starts_at BETWEEN ? AND ?) OR (all_day IS TRUE AND ends_at BETWEEN ? AND ?)',
                                       date.beginning_of_day, date.end_of_day,
                                       date.beginning_of_day, date.end_of_day) }
+  scope :between, -> (start, stop) { where('(starts_at BETWEEN ? AND ?) OR (all_day IS TRUE AND ends_at BETWEEN ? AND ?)',
```

Comment:
Line is too long. [122/100]<br>Block body expression is on the same line as the block start.